app/api/utility.py

```diff
@@ -24,21 +57,33 @@ def parse_nodes_as_dict(path: Path) -> dict:
     where the keys are the node URLs, and the values are the node names.
     Makes sure node URLs end with a slash.
     """
```

Comment:
🍒: we could update the docstring to specify that only valid nodes will be returned